velox/exec/tests/WindowTest.cpp

```diff
@@ -182,6 +76,48 @@ TEST_F(WindowTest, spill) {
   ASSERT_GT(stats.spilledPartitions, 0);
 }
 
+TEST_F(WindowTest, duplicateOrOverlappingKeys) {
```

Comment:
Can we move this after missingFunctionSignatures so that we don't see this diff.